src/foam/core/XML.js

```diff
@@ -444,21 +453,24 @@ foam.LIB({
   constants: {
     // Pretty Print
     Pretty: foam.xml.Outputter.create({
-      outputDefaultValues: false
+      outputDefaultValues: false,
+      outputDefinedValues: true
     }),
 
     // Compact output (not pretty)
     Compact: foam.xml.Outputter.create({
       pretty: false,
       formatDatesAsNumbers: true,
       outputDefaultValues: false,
+      outputDefinedValues: true
```

Comment:
I think this should be false. Compact is meant to save space, so it doesn't make sense to output values which are set the the default.